x-pack/solutions/security/plugins/security_solution/public/timelines/components/bottom_bar/index.tsx

```diff
@@ -65,7 +65,7 @@ export const TimelineBottomBar = React.memo<TimelineBottomBarProps>(
           </EuiFlexItem>
           {!show && ( // this is a hack because TimelineEventsCountBadge is using react-reverse-portal so the component which is used in multiple places cannot be visible in multiple places at the same time
             <EuiFlexItem grow={false} data-test-subj="timeline-event-count-badge">
-              <TimelineEventsCountBadge />
+              <TimelineQueryTabEventsCount timelineId={timelineId} />
```

Comment:
Are you planning to remove that `OutPortal` `InPortal` stuff as well in this PR?

Reply:
No, I wasn't planning on making that change in this PR